pkg/blobstore/readfallback/read_fallback_blob_access_test.go

```diff
@@ -20,7 +21,8 @@ func TestReadFallbackBlobAccessGet(t *testing.T) {
 
 	primary := mock.NewMockBlobAccess(ctrl)
 	secondary := mock.NewMockBlobAccess(ctrl)
-	blobAccess := blobstore.NewReadFallbackBlobAccess(primary, secondary)
+	replicator := replication.NewNoopBlobReplicator(secondary)
```

Comment:
Please use a `replicator := mock.NewBlobReplicator(ctrl)` here. NoopBlobReplicator is not the type we should be testing here.

Once you adjust all of the tests below, you can remove all of `TestReadFallbackBlobAccessReplication()`, as `TestReadFallbackBlobAccessGet()` will then provide sufficient coverage.

Reply:
Ah yes, artifact of the previous iteration.